### PR TITLE
Change RWD to RWD Principles to prevent collapsible menu conflict

### LINKED ISSUE #12731: [beta] "Responsive Web Design" Sub list has duplicate attribute
#### Challenge Name

http://beta.freecodecamp.com/en/map

#### Issue Description

If you collapse or expand the "Responsive Web Design" sub-list from the map list, it expands or collapses the whole Responsive Web Design List. Because it is targeting the same id.

### PR DESCRIPTION
**Pull request by dhcodes**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of freeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #12731

#### Description

Because the Responsive Web Design subcategory has the same name as the main heading (and thus the same id), clicking the collapsible menu for the submenu would collapse the main category. I renamed the subcategory to "Responsive Web Design Principles" to prevent this conflict.

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 1]
```diff
@@ -872,7 +872,7 @@
         "capitalizeString(\"hello!\");"
       ],
       "tests": [
-        "assert(code.match(/import\\s+\\{\\s?capitalizeString\\s?\\}\\s+from\\s+\"string_functions\"/ig)"
+        "assert(code.match(/import\\s*\\{\\s*capitalizeString\\s*\\}\\s*from\\s*\\\"string_functions\\\"/ig), 'message: Import <code>capitalizeString</code> function from the <code>\"string_functions\"</code> file');"
```
Comment (systimotic) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Was this added accidentally? Somewhat related https://github.com/freeCodeCamp/freeCodeCamp/pull/12745
Name change LGTM 👍 Should the file name also be changed?

Comment (dhcodes) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: Yes, was working on that and must have been on the wrong branch. I'll update this tonight. 

I don't think the filename needs to be changed.
Thanks @systimotic

**Comment by dhcodes** [comment]
@systimotic this is fixed now.

**Review by systimotic — APPROVED** [review]
# LGTM 👍 👍 
Nice work Dylan 🙂

**Pull request merged**